### PR TITLE
Improve user list

### PR DESCRIPTION
**Pull request by binh-dam-ibigroup**
### Checklist

- [x] Appropriate branch selected => requires merging branch `user-account-types` to `dev` first.
- [x] Any modified or new methods or classes have helpful JSDoc and code is thoroughly commented
- [na] The description lists all applicable issues this PR seeks to resolve
- [x] The description lists any configuration setting(s) that differ from the default settings
- [x] All tests and CI builds passing

### Description

Blocked by/requires #855. 

This PR improves the list of datatools users in the admin section by converting it to a table and adding info such as account type and last login. App admin badges are now under the account type column. Projects are under the project column The layout of commands while editing a user is changed slightly to accommodate the new user info shown.

Screen Shot 2022-09-28 at 8 38 19 AM

### CODE REVIEW

File: lib/admin/components/UserRow.js [hunk 1]
```diff
@@ -154,78 +151,120 @@ export default class UserRow extends Component<Props, State> {
...
+            {!permissions.isApplicationAdmin() && this._getUserPermissionLabel(permissions)}
+            {' '}
+            {this._getOrgLabel(permissions)}
+          </td>
+          {/* When not editing, show the "Last login" cell and
+              show the "actions" column with just the edit button.
+              When editing, hide the "Last login" cell and
+              expand the "actions" column over the space where last login is shown,
+              so that there is enough space to show the delete/save/cancel buttons. */}
+          <td style={isEditing ? { display: 'none' } : null}>
```
Comment (miles-grant-ibigroup) [inline lib/admin/components/UserRow.js]: Why do we use `display: none` instead of just not rendering the element?

**Review by miles-grant-ibigroup — APPROVED** [review]
Looks and works great! Thanks so much. Just one quick question about a small implementation detail, but not blocking.

File: lib/admin/components/UserRow.js (hunk 1 above)
Comment (binh-dam-ibigroup) [reply lib/admin/components/UserRow.js]: My intuition was to keep the DOM tree the same instead of adding/removing an element on toggle.

Comment (miles-grant-ibigroup) [reply lib/admin/components/UserRow.js]: Wouldn't removing the element be the more "react-ey" way to do it? There may be performance implications but I think within datatools these are ignorable.

Comment (binh-dam-ibigroup) [reply lib/admin/components/UserRow.js]: Updated in https://github.com/ibi-group/datatools-ui/pull/859/commits/60b036c9179c7974ca48bf4c5c7087deb2fe05fa.

**Review by philip-cline — APPROVED** [review]

**Pull request merged**